properties/P000208.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: P000208
+name: Noetherian
+aliases:
+  - Hereditarily compact
+refs:
+  - wikipedia: Noetherian_topological_space
+    name: Noetherian topological space
+  - zb: "1325.14001"
+    name: Algebraic Geometry II (Mumford-Oda)
+  - doi: 10.1017/9781316543870
+    name: Spectral spaces (Dickmann, Schwartz, Tressl)
+---
+
+A space for which every subspace is compact.
+
+Equivalently:
+- Every open subset is compact.
+- The open sets satisfy the ascending chain condition: There is no infinite strictly increasing chain $O_1 \subsetneq O_2 \subsetneq \cdots$ of open sets.
+- The closed sets satisfy the descending chain condition: There is no infinite strictly decreasing chain $Y_1 \supsetneq Y_2 \supsetneq \cdots$ of closed sets.
```

Comment:
```suggestion
- The open sets satisfy the *ascending chain condition*: There is no infinite strictly increasing sequence $O_1 \subsetneq O_2 \subsetneq \cdots$ of open sets.
- The closed sets satisfy the *descending chain condition*: There is no infinite strictly decreasing sequence $Y_1 \supsetneq Y_2 \supsetneq \cdots$ of closed sets.
```

Reply:
Sorry, I delete my previous comment and added it back as suggestion.

Reply:
I accidentally made the first change before seeing the suggestion.